gitogether/1st-meeting/isfha.html

```diff
@@ -6,6 +6,9 @@
 </head>
 <body>
 	<main>
+		<figure>
+			<img src="https://user-images.githubusercontent.com/13671268/66139346-7821b280-e62a-11e9-8439-a204be6da13d.jpeg" alt="avatar">
```

Comment:
invalid URL, please change it with another URL, thanks!!!!

Reply:
Ok i understand your feeling, please review my latest commit.